lib/versioning/VersioningRequestProcessor.js

```diff
@@ -50,8 +50,8 @@ class VersioningRequestProcessor {
         this.repairing = {};
     }
 
-    generateVersionId() {
-        return genVID(this.uidCounter++);
+    generateVersionId(replicationGroupToken) {
+        return genVID(this.uidCounter++, replicationGroupToken);
```

Comment:
This is Arsenal, how to you intend to get the value from S3 or Metadata config without passing it along through the API? Do you mean there is an Arsenal-specific config we can use so we can import the value directly?